### ISSUE TITLE
ImageNet pretrained ResNeSt-269 for detectron2

### ISSUE THREAD

**Issue by zzzxxxttt** [chongruo/detectron2-ResNeSt#34]
I would like to train a Mask-RCNN with ResNeSt-269 backbone, could you provide the ImageNet pretrained ResNeSt-269 model?

**Comment by chongruo** [comment]
Here are all pre-trained models

```
https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest50_detectron-255b5649.pth
https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest101_detectron-486f69a8.pth
https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest200_detectron-02644020.pth
https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest269_detectron-69df4066.pth
```

**Comment by zzzxxxttt** [comment]
That helps a lot, thank you very much!

**Comment by tonysy** [comment]
@chongruo  Hi, could you share the recipe to get such pre-trained model with the classification head. I want to check the classification performance of the pre-trained model. Thanks a lot.

**Comment by zhanghang1989** [comment]
> Here are all pre-trained models
> 
> ```
> https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest50_detectron-255b5649.pth
> https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest101_detectron-486f69a8.pth
> https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest200_detectron-02644020.pth
> https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnest269_detectron-69df4066.pth
> ```

Updating the URLs:

https://s3.us-west-1.wasabisys.com/resnest/detectron/resnest50_detectron-255b5649.pth
https://s3.us-west-1.wasabisys.com/resnest/detectron/resnest101_detectron-486f69a8.pth
https://s3.us-west-1.wasabisys.com/resnest/detectron/resnest200_detectron-02644020.pth
https://s3.us-west-1.wasabisys.com/resnest/detectron/resnest269_detectron-69df4066.pth
